### PR TITLE
feature/e-commerce template Jordi Pujol

### PR DESCRIPTION
**Pull request by JordiRoronoa**

### CODE REVIEW

**Comment by gcelma** [comment]
Maybe the header could be higher in the tablet and laptop version...

File: staff/jordi-pujol/e-commerce-template/style.css [hunk 1]
```diff
@@ -0,0 +1,96 @@
+*{
+    box-sizing: border-box;
+}
+
+body{
+    margin: 0px;
```
Comment (maryammdot) [inline staff/jordi-pujol/e-commerce-template/style.css]: when you give margin 0, you dont need to add px. You can write margin: 0;

File: staff/jordi-pujol/e-commerce-template/style.css [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+    width: 400px;
+    height: 250px;
+}
+
+/* MEDIA QUERY - TABLET ------------------------ */
+
+@media (min-width: 768px) {
+    main {
+      display: grid;
+      grid-template-columns: 1fr 1fr;
```
Comment (maryammdot) [inline staff/jordi-pujol/e-commerce-template/style.css]: you could use repeat(2, 1fr).

File: staff/jordi-pujol/e-commerce-template/style.css (hunk 1 above)
Comment (JordiRoronoa) [reply staff/jordi-pujol/e-commerce-template/style.css]: I've just changed it! thanks for the feedback.

File: staff/jordi-pujol/e-commerce-template/style.css (hunk 2 above)
Comment (JordiRoronoa) [reply staff/jordi-pujol/e-commerce-template/style.css]: Done! thanks for the advice

**Pull request merged**